src/model/resources/curtailable_variable_renewable/curtailable_variable_renewable.jl

```diff
@@ -39,12 +39,20 @@ function curtailable_variable_renewable!(EP::Model, inputs::Dict, setup::Dict)
 	@expression(EP, ePowerBalanceDisp[t=1:T, z=1:Z],
 	sum(EP[:vP][y,t] for y in intersect(VRE, dfGen[dfGen[!,:Zone].==z,:R_ID])))
 
-	EP[:ePowerBalance] += ePowerBalanceDisp
+	for t=1:T
+		for z=1:Z
+			add_to_expression!(EP[:ePowerBalance][t, z], ePowerBalanceDisp[t, z])
+		end
+	end
```

Comment:
Can we add methods to `add_to_expression!` that takes two integers in order to avoid an explicit 5-line for-loop?
This could ensure that the loop order (row-major vs column-major) is correct.